### PR TITLE
add popup to package

### PR DESCRIPTION
**Pull request by UB3RL33T**

### CODE REVIEW

File: i18n/de_DE.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+"We have noticed that you are visiting this website from ""%1"".","Wir haben festgestellt, dass Sie diese Webseite aus ""%1"" besuchen"
+"If we have set the country incorrectly, please change the country using our country list (at the top right of the screen).","Sollten wir das Land falsch festgelegt haben, dann ändern Sie das Land über unsere Länderliste (am oberen rechten Bildschirmrand)."
+"Thank you very much and enjoy your visit in the SATA-Shop.","Vielen Dank und viel Freude bei Ihrem Einkauf im SATA-Shop"
```
Comment (DanieliMi) [inline i18n/de_DE.csv]: Nope. Customer name shouldn't be in a puplic package!

File: i18n/es_ES.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+"We have noticed that you are visiting this website from ""%1"".","Hemos notado que usted está visitando esta página web desde ""%1""."
+"If we have set the country incorrectly, please change the country using our country list (at the top right of the screen).","Si hemos configurado el país de forma incorrecta, cambie el país utilizando nuestra lista de países (en la parte superior derecha de la pantalla)."
+"Thank you very much and enjoy your visit in the SATA-Shop.","Muchas gracias y disfrute de su compra en la tienda SATA."
```
Comment (DanieliMi) [inline i18n/es_ES.csv]: See above.

File: i18n/fr_FR.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+"We have noticed that you are visiting this website from ""%1"".","Nous avons remarqué que vous visitez ce site depuis ""%1""."
+"If we have set the country incorrectly, please change the country using our country list (at the top right of the screen).","Si nous avons mal réglé le pays, veuillez changer de pays à l'aide de notre liste de pays (en haut à droite de l'écran)."
+"Thank you very much and enjoy your visit in the SATA-Shop.","Merci beaucoup et bon achat dans le SATA-Shop."
```
Comment (DanieliMi) [inline i18n/fr_FR.csv]: See above.

File: i18n/it_IT.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+"We have noticed that you are visiting this website from ""%1"".","Abbiamo notato che state visitando questo sito web dalla ""%1""."
+"If we have set the country incorrectly, please change the country using our country list (at the top right of the screen).","Se abbiamo impostato il paese in modo errato, si prega di cambiare il paese utilizzando il nostro elenco dei paesi (in alto a destra dello schermo)."
+"Thank you very much and enjoy your visit in the SATA-Shop.","Grazie mille e godetevi l'acquisto nel SATA-Shop."
```
Comment (DanieliMi) [inline i18n/it_IT.csv]: See above.

File: view/frontend/templates/popup.phtml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+if ($viewModel->isModuleActive()): ?>
+
+    <div id="store-switcher-popup" style="display:none;">
+        <?php //FIXME use different message for IP not found?
+        ?>
+        <p><?= __('We have noticed that you are visiting this website from "%1".',
+                $viewModel->getCountryNameFromIp()) ?></p>
+        <p><?= __('If we have set the country incorrectly, please change the country using our country list (at the top right of the screen).') ?>
+            .</p>
+        <p><?= __('Thank you very much and enjoy your visit in the SATA-Shop.') ?></p>
```
Comment (DanieliMi) [inline view/frontend/templates/popup.phtml]: See above comments.

**Review by DanieliMi — CHANGES_REQUESTED** [review]
See comments

File: i18n/de_DE.csv (hunk 1 above)
Comment (UB3RL33T) [reply i18n/de_DE.csv]: fixed

File: i18n/es_ES.csv (hunk 2 above)
Comment (UB3RL33T) [reply i18n/es_ES.csv]: fixed

**Comment by UB3RL33T** [comment]
Merging in accordance with @DanieliMi

**Pull request merged**